massbalancemachine/models/CustomXGBoostRegressor.py

```diff
@@ -92,8 +92,6 @@ def randomsearch(
         splits: Dict[str, Union[list, np.ndarray]],
         features: pd.DataFrame,
         targets: np.ndarray,
-        num_jobs: int = None,
```

Comment:
I left this in because it allows the users a quick way to change the parameters and try stuff without re-importing the config file (which is sometimes a bit cumbersome). What do you think about this?

Reply:
Ah, I thought it was something that was just forgotten, but seeing how it's the number of jobs and seed, the notebook generally needs to be reloaded anyway as (at least the seed) also affects other processes. I think it would make sense for other parameters but might leed to confusion here for these two.

Reply:
Ah okay, clear! And makes sense indeed.